solpolpy/alpha.py

```diff
@@ -27,9 +27,8 @@ def radial_north(shape):
     x = np.arange(-x_size // 2, x_size // 2)
     y = np.arange(-y_size // 2, y_size // 2)
     xx, yy = np.meshgrid(x, y)
-    # return np.fliplr(np.arctan2(yy, xx))*u.radian
-    return np.flipud(np.rot90(np.fliplr(np.arctan2(yy, xx) + np.pi), k=1))* u.radian
-
+    return np.rot90(np.fliplr(np.arctan2(yy, xx)+np.pi), k=1)*u.radian
```

Comment:
This is not covered by tests. I notice most of the alpha.py file isn't. Let's write some tests and also make proper docstrings.